guides/source/threading_and_code_execution.md

```diff
@@ -247,7 +400,7 @@ block, and autoload knows when to upgrade to a `load` lock, and switch back to
 
 Other blocking operations performed inside the Executor block (which includes
 all application code), however, can needlessly retain the `running` lock. If
-another thread encounters a constant it must autoload, this can cause a
+another thread encounters a constant it must autoload, which can cause a
```

Comment:
Current wording is correct; maybe "a constant _that_ it must autoload" would have prevented the misread?